docs/books/admin_guide/09-backups.md

```diff
@@ -463,13 +463,13 @@ tar xvfP /backups/etc.133.P.tar
 
 !!! Warning
 
-    Go to the right place.
+    For security reasons, caution should be exercised when extracting backup files saved in absolute mode.
```

Comment:
Again, you are introducing passive voice here. An acceptable rewording would be:

"For security reasons, you should use caution when extracting backup files, saved in absolute mode."